src/dynamic_honey_badger/builder.rs

```diff
@@ -58,26 +60,42 @@ where
         self
     }
 
+    pub fn subset_handling_strategy(
```

Comment:
Please add a short doc comment explaining subset handling. (Also in `HoneyBadgerBuilder`.)

Reply:
I noticed I was missing documentation and intended to get to it today -- thank you for reminding me! :)